adam/language_specific/english/english_language_generator.py

```diff
@@ -331,6 +390,54 @@ def _preposition_for_region_as_goal(self, region: Region[SituationObject]) -> st
                     f"Don't know how to translate {region} to a preposition yet"
                 )
 
+        def _translate_relation_to_verb(
+            self, relation: Relation[SituationObject]
+        ) -> DependencyTreeToken:
+            lexicon_entry = self._unique_lexicon_entry(relation.relation_type)
+            if any(
```

Comment:
Is there similar logic for other verbs that we can refactor into a method?

Reply:
That's slightly different because we access those through argument roles.

Reply:
Got it.